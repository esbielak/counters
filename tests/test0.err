ERROR 21
ERROR 31
