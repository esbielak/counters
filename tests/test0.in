M 0 5 11
A 3
P 0
A 5
P 0
A 4
P 0
A 49
P 0
A 5
P 0
A 1
P 0
A 6
P 0
A 0
P 0
A 200
P 0
D 0
D 0
F 0 0
A 1
P 0
A 10
P 0
A 30
P 0
A 0
P 0
G 0 1
G 1 0
A 3
P 1
A 10
P 1
A 1000
P 1
A 100
P 1
P 0
