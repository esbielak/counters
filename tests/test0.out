C 0 1
C 0 2
C 0 2
C 0 11
C 0 11
E 0 1
C 0 0
C 0 1
C 0 1
E 0 66
E 0 138
C 0 10
E 0 1
C 0 1
E 0 1
E 0 2
E 0 4
E 0 7
C 0 11
E 0 2
E 0 10
E 0 23
C 0 41
C 0 41
C 1 3
E 1 7
C 1 3
E 0 1
E 0 35
E 0 90
E 1 97
E 0 179
E 0 323
E 0 556
E 0 933
C 1 903
E 1 97
C 1 3
C 0 1154
